scripts/drupal.sh

```diff
@@ -82,13 +82,19 @@ $DRUSH_CMD -y en devel
 $DRUSH_CMD -y pm-uninstall search
 $DRUSH_CMD en -y search_api
 
-# Set default theme to bootstrap
-$DRUSH_CMD -y en bootstrap
-$DRUSH_CMD -y config-set system.theme default bootstrap
-## THIS IS STUPID
-# blocks are tied to themes because themes define the regions available to place blocks. gross.
-##
-
+# Set default theme to carapace (and download dependencies, will composer-ize later)
+$DRUSH_CMD --pm-force dl -y adaptivetheme
+$DRUSH_CMD --pm-force dl -y at_tools
+$DRUSH_CMD --pm-force dl -y layout_plugin
```

Comment:
So these 3 lines will have to go into [drupal-project](https://github.com/Islandora-CLAW/drupal-project) eventually, but you can replace them here with
```
cd $DRUPAL_HOME
composer require "drupal/adaptivetheme:^2.0" "drupal/at_tools:^2.0" "drupal/layout_plugin:^1.0@alpha"
```
For reference I figured out the numbers from the individual project pages on drupal.org (removing the `8.x-`)

I'll check out @Favenzio's theme next 😉